ui/components/TracePage/Table.tsx

```diff
@@ -13,15 +14,21 @@ export const Table = ({
   rows: StringToScalar[];
   rowIds?: string[];
   onFocusChange?: (rowId?: string) => void;
-}) => (
-  <AgGridReact<StringToScalar>
-    className="ag-theme-alpine w-full h-full"
-    columnDefs={uniq(rows.flatMap(Object.keys)).map(field => ({ field }))}
-    defaultColDef={{ filter: true, sortable: true, resizable: true }}
-    onCellFocused={({ rowIndex }) =>
-      onFocusChange?.(rowIndex === null ? undefined : rowIds?.[rowIndex])
-    }
-    pagination={rows.length > 100}
-    rowData={rows}
-  ></AgGridReact>
-);
+}) => {
+  const pagination = rows.length > 100;
+  return rows.every(isEmpty) ? null : (
```

Comment:
No explanation text?

Reply:
I originally missed your comment about it but then noticed it later and just added text.

(I really wish that root-level comments also required resolution.)